tests/unit/store/traction/pacbio/plates/actions.spec.js

```diff
@@ -1,98 +1,57 @@
   })
 
   describe('setPlates', () => {
     it('fetches the plates from the service, and commits them', async () => {
-      get.mockReturnValue(Data.PacbioPlatesRequest)
+      get.mockReturnValue(successResponse)
       const { success, errors } = await Actions.setPlates({ commit, getters })
 
-      expect(commit).toHaveBeenCalledWith('setPlates', expectedPlates)
+      expect(commit).toHaveBeenCalledWith('setPlates', pacbioPlatesRequestFactory.content.data)
       expect(success).toEqual(true)
-      expect(errors).toEqual([])
+      expect(errors).toEqual({})
     })
 
     it('errors fetching the plates', async () => {
-      get.mockRejectedValue({ response: failedResponse })
+      get.mockRejectedValue(failedResponse)
 
       const { success, errors } = await Actions.setPlates({ commit, getters })
 
       expect(commit).not.toHaveBeenCalled()
       expect(success).toEqual(false)
-      expect(errors).toEqual('error1 There was an error')
+      expect(errors.error1).toEqual(['There was an error'])
     })
   })
 
   describe('findPlate', () => {
     it('fetches the plate from the service, and returns it with wells and requests', async () => {
-      get.mockReturnValue(Data.PacbioPlatesRequest)
-      const expectedPlate = {
-        id: '61',
-        barcode: 'DN814327C',
-        created_at: '2021/06/03 06:59',
-        wells: [
-          {
-            position: 'A1',
-            requests: [
-              {
-                id: '40',
-                library_type: null,
-                estimate_of_gb_required: null,
-                number_of_smrt_cells: null,
-                cost_code: null,
-                external_study_id: 'fec8a1fa-b9e2-11e9-9123-fa163e99b035',
-                sample_name: 'sample_DN814327C_A1',
-                barcode: null,
-                sample_species: 'human',
-                created_at: '2021/06/03 06:59',
-                source_identifier: 'DN814327C:A1',
-              },
-            ],
-          },
-          {
-            position: 'A2',
-            requests: [
-              {
-                id: '41',
-                library_type: null,
-                estimate_of_gb_required: null,
-                number_of_smrt_cells: null,
-                cost_code: null,
-                external_study_id: 'fec8a1fa-b9e2-11e9-9123-fa163e99b035',
-                sample_name: 'sample_DN814327C_A2',
-                barcode: null,
-                sample_species: 'human',
-                created_at: '2021/06/03 06:59',
-                source_identifier: 'DN814327C:A2',
-              },
-            ],
-          },
-        ],
-      }
+      get.mockReturnValue(successResponse)
+
       // Barcode provided is first plate in Data.PacbioPlatesRequest
       const plate = await Actions.findPlate({ commit, getters }, { barcode: 'DN814327C' })
 
-      expect(plate).toEqual(expectedPlate)
+      expect(plate).toEqual(pacbioPlatesRequestFactory.expectedPlates)
```

Comment:
I did something similar but used the storeData and used storeData.selected etc. Might be worth following a similar model.

Reply:
This actually requires a method to do the same kind of data restructuring as it does in actions to reformat the plate requests to well. So, i don't know how useful it will be because we end up replicating similar code that we intend to test.